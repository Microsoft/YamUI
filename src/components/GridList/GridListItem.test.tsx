/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { GridListItem, BaseComponentProps } from './index';

describe('<GridList />', () => {
  let component: ShallowWrapper<BaseComponentProps>;

  describe('without props', () => {
    beforeEach(() => {
      component = shallow(<GridListItem>content</GridListItem>);
    });

    it('matches its snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });

  describe('with additional className', () => {
    beforeEach(() => {
      component = shallow(<GridListItem className="TEST_CLASSNAME">content</GridListItem>);
    });

    it('matches its snapshot', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
