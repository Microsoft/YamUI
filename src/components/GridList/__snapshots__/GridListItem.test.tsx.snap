// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridList /> with additional className matches its snapshot 1`] = `
<li
  className="y-gridListItem TEST_CLASSNAME"
>
  <div>
    content
  </div>
</li>
`;

exports[`<GridList /> without props matches its snapshot 1`] = `
<li
  className="y-gridListItem"
>
  <div>
    content
  </div>
</li>
`;
